wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,343 @@
+The `#spanish` chat channel is for those who speak Spanish.
+```
+
+### Proper nouns
+
+Proper nouns must be capitalised. The following is an example:
+
+```markdown
+Dean Herbert (also known as peppy) created osu! back in 2007.
+```
+
+### Trademarks
+
+*For rules regarding osu!, see: [osu!](#osu!)*
+
+The following trademarks must be spelt as follows (letter casing must match):
+
+- `Discord`
+- `Facebook`
+- `GitHub`
+- `Google`
+- `Reddit`
+- `Skype`
+- `Twitch`
+- `Twitter`
+- `YouTube`
+
+Trademarks must not be followed by the trademark or registered trademark symbols.
+
+## Date and time
+
+### Date formatting
+
+*For date formatting, see [Wikipedia's Manual of Style on Date formatting](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats)*
+
+Date formatting should be consistent throughout the entire article.
+
+### Time formatting
+
+Time formatting must be done using the following format:
+
+```markdown
+HH:MM TIMEZONE
+```
+
+Time is written in 24-hour format and any single digits must be prefixed with a zero. The timezone must be written immediately after the time and should be in UTC+0 for global events or in the timezone of the event for smaller events. Use `UTC` instead of `GMT`.
+
+Bad examples:
+
+```markdown
+3:30 PM UTC
+22:30 (UTC+7)
+11:30
+```
+
+Good examples:
+
+```markdown
+15:30 UTC
+22:30 UTC+7
+11:30 UTC-4
+```
+
+`UTC` (without an offset) is implied to be `UTC+0`. Either one may be used, but usage must be consistent. UTC with a 0 offset must use a plus symbol (`+`).
+
+### Date and time formatting
+
+When both date and time are used together, the date must be written first, followed by the time. The time must be in between parenthesis (`(` and `)`). The following are examples:
+
+Bad examples:
+
+```markdown
+October 25, 2016 at 11:45 UTC
+October 25, 2016 11:45 UTC
+```
+
+Good example:
+
+```markdown
+October 25, 2016 (11:45 UTC)
+```
+
+## Game modes
+
+Game modes must be written as follows:
+
+- `osu!standard` (unofficial, but used to prevent ambiguity)
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+*Notice: `osu!standard` is used to maintain consistency when referring to the game mode. Folder names, however, must use `osu!` even if it is referring to the game mode.*
+
+Referring to the old game mode names (i.e. `Catch the Beat`, `Taiko`, and `Mania`) may be done if discussing said game mode's previous name.
+
+---
+
+Articles such as the [Ranking Criteria](/wiki/Ranking_Criteria) may use `osu!` instead of `osu!standard`.
+
+## osu!
+
+The name of the game, osu!, must not be capitalised or italicised. The osu! official branding must not use any spaces. Examples include:
+
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
+- `osu!direct`
+- `osu!keyboard`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!tablet`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
+
+Other brandings that are not covered by the osu! official branding must use a space. Examples include:
```

Comment:
I think it may be described better if you also point out that `osu!` here is a [qualifying noun](https://en.wikipedia.org/wiki/Noun_adjunct) (which should be separated by a space from the noun it modifies), and not a part of some specific term like *`osu!`supporter*